packages/gatsby/src/bootstrap/resolve-module-exports.js

```diff
@@ -3,7 +3,7 @@ const fs = require(`fs`)
 const traverse = require(`@babel/traverse`).default
 const get = require(`lodash/get`)
 const { codeFrameColumns } = require(`@babel/code-frame`)
-const { babelParseToAst } = require(`../utils/babel-parse-to-ast`)
+import { babelParseToAst } from "../utils/babel-parse-to-ast"
```

Comment:
Why isn't import used for others ?

Reply:
Thanks, I've changed other statements to use the `import` syntax instead of `require`.